src/hb-subset-input.cc

```diff
@@ -448,22 +448,30 @@ hb_subset_input_pin_axis_location (hb_subset_input_t  *input,
 #endif
 #endif
 
-#ifdef HB_EXPERIMENTAL_API
 /**
  * hb_subset_preprocess
  * @input: a #hb_face_t object.
  *
  * Preprocesses the face and attaches data that will be needed by the
  * subsetter. Future subsetting operations can then use the precomputed data
- * to speed up the subsetting operation.
+ * to speed up the subsetting operation. Returns a new hb_face_t*.
```

Comment:
This should be on a line of its own (above `Since:`) with a colon after Returns.